<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="appContainer" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1200</width>
     <height>800</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#appContainer {
	background-color: rgb(239, 239, 239);
}


QGroupBox {
 	border: none;
	font: 75 14pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton {
	font: 75 14pt &quot;MS Shell Dlg 2&quot;;
}</string>
   </property>
   <widget class="QGroupBox" name="ProtocolSelection">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>471</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Protocol Selection</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>441</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="ProtocolSelectionConctainer" stretch="0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="ProtocolSelectionLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Protocol Selection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ProtocolSelectionInput">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>IP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DNS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ARP</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="ProtocolsContainer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>491</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_2">
     <widget class="QGroupBox" name="IPProtocolDetails">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>471</width>
        <height>511</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>IP Protocol Details</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_23">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>441</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="SourceMacAddressContainer_4" stretch="0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="SourceMacAddressLabel_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Source IP Address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="SourceMacAddressInput_8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>00:11:22:33:44:55</string>
          </property>
          <property name="MAC_ADDRESS" stdset="0">
           <bool>false</bool>
          </property>
          <property name="IP" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_24">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>441</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="DestinationMacAddressContainer_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="DestinationMacAddressLabel_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Destination IP Address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="SourceMacAddressInput_9">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
          <property name="MAC_ADDRESS" stdset="0">
           <bool>false</bool>
          </property>
          <property name="IP" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_25">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>439</width>
         <height>39</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="PayloadContainer_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="PayloadLabel_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Payload</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="PayloadInput_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Payload</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_1">
     <widget class="QGroupBox" name="IPProtocolDetails_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>471</width>
        <height>511</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>DNS Protocol Details</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_26">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>441</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="SourceMacAddressContainer_5" stretch="0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="SourceMacAddressLabel_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Source MAC Address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="SourceMacAddressInput_10">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>00:11:22:33:44:55</string>
          </property>
          <property name="MAC_ADDRESS" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_27">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>441</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="DestinationMacAddressContainer_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="DestinationMacAddressLabel_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Destination MAC Address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="SourceMacAddressInput_11">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>00:11:22:33:44:55</string>
          </property>
          <property name="MAC_ADDRESS" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QGroupBox" name="ARPProtocolDetails">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>471</width>
        <height>511</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>ARP Protocol Details</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_32">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>441</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="SourceMacAddressContainer_7" stretch="0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="SourceMacAddressLabel_7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Source MAC Address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="SourceMacAddressInput_12">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>00:11:22:33:44:55</string>
          </property>
          <property name="MAC_ADDRESS" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_33">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>441</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="DestinationMacAddressContainer_7">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="DestinationMacAddressLabel_7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Destination MAC Address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="SourceMacAddressInput_13">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>00:11:22:33:44:55</string>
          </property>
          <property name="MAC_ADDRESS" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="SendPacketButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>491</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Packet</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
